src/main/java/rx/internal/util/ObjectPool.java

```diff
@@ -54,11 +49,11 @@ private ObjectPool(final int min, final int max, final long validationInterval)
         // initialize pool
         initialize(min);
 
-        schedulerWorker = Schedulers.computation().createWorker();
```

Comment:
I'm not convinced about this being better or worse. What data is driving this decision? It is unlikely this has anything to do with the numbers being tested in this PR with changes to `observeOn`.